__tests__/app.test.js

```diff
@@ -249,4 +245,61 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("PATCH-api/articles/:article_id", () => {
+    test("200-endpoint increases/decreases vote count of specified article by the quantity included in the request body", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then((results) => {
+          expect(results.body.article).toMatchObject({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 101,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
```

Comment:
You could add in a test to check that the votes are decremented, like this: `{ inc_votes : -100 }`